static/css/_vars.scss

```diff
@@ -7,6 +7,12 @@ $links-light: #80CDEF;
 $secondary: #A7A9AC;
 $secondary-light: #F3F3F3;
 $main: black;
+$green: #8EBE3F;
+$green-light: #E3EFCF;
+$blue:  #009BDF;
+$blue-light: #80CDEF;
+$red: #E31D3C;
+$red-light: #F8C7CE;
```

Comment:
Instead of `<color>-light`, use `<color>--light` (two dashes). It's the more proper way of doing it.